### PR TITLE
Suggested fixes and consistency fixes

### PR DESCRIPTION
**Pull request by 2byrds**
Applied several suggested changes from @swcurran , migrated to a consistent use of the term key state, and stubbed-out some service endpoint info per our meeting today.
I'll do the keri.cesr fix in a separate PR.

### CODE REVIEW

File: spec/diddocuments.md [hunk 1]
```diff
@@ -223,7 +223,11 @@ KERI AID public keys can be used to sign a variety of data.  This includes but i
 References to verification methods in the DID document MUST use the relative form of the identifier, e.g., `"authentication": ["#<identifier>"]`.
 
 #### Service Endpoints
-In KERI, service endpoints are defined by 2 sets of signed data that follow the Best Available Data - Read, Update, Nullify ([[ref: BADA-RUN]]) rules for data processing.  The protocol ensures that all data is signed in transport and at rest and versioned to ensure only the latest signed data is available.
+Did:webs supports service endpoint types declared in the DID Specification Registries, such as [DIDCommMessaging](https://www.w3.org/TR/did-spec-registries/#didcommmessaging).  For additional details about the mapping between KERI events and the Service Endpoints in the DID Document, see [Service Endpoint KERI events](#service-endpoint-event-details).  It is important to note that DID document service endpoints are different than the KERI service endpoints detailed in [KERI Service Endpoints as DID Document Metadata](#keri-service-endpoints-as-did-document-metadata).
```
Comment (peacekeeper) [inline spec/diddocuments.md:226]:
```suggestion
Did:webs supports service endpoints, including types declared in the DID Specification Registries, such as [DIDCommMessaging](https://www.w3.org/TR/did-spec-registries/#didcommmessaging).  For additional details about the mapping between KERI events and the Service Endpoints in the DID Document, see [Service Endpoint KERI events](#service-endpoint-event-details).  It is important to note that DID document service endpoints are different than the KERI service endpoints detailed in [KERI Service Endpoints as DID Document Metadata](#keri-service-endpoints-as-did-document-metadata). 
```

**Review by peacekeeper — APPROVED** [review]
Looks good, just one small suggestion. It would be strange if did:webs supported ONLY those service types that are in the DID Spec Registries, to my knowledge no other DID method imposes such a restriction.

File: spec/diddocuments.md [hunk 2]
```diff
@@ -223,7 +223,7 @@ KERI AID public keys can be used to sign a variety of data.  This includes but i
...
 
 #### Service Endpoints
-Did:webs supports service endpoint types declared in the DID Specification Registries, such as [DIDCommMessaging](https://www.w3.org/TR/did-spec-registries/#didcommmessaging).  For additional details about the mapping between KERI events and the Service Endpoints in the DID Document, see [Service Endpoint KERI events](#service-endpoint-event-details).  It is important to note that DID document service endpoints are different than the KERI service endpoints detailed in [KERI Service Endpoints as DID Document Metadata](#keri-service-endpoints-as-did-document-metadata). 
+Did:webs supports service endpoints, including types declared in the DID Specification Registries, such as [DIDCommMessaging](https://www.w3.org/TR/did-spec-registries/#didcommmessaging).  For additional details about the mapping between KERI events and the Service Endpoints in the DID Document, see [Service Endpoint KERI events](#service-endpoint-event-details).  It is important to note that DID document service endpoints are different than the KERI service endpoints detailed in [KERI Service Endpoints as DID Document Metadata](#keri-service-endpoints-as-did-document-metadata).
```
Comment (swcurran) [inline spec/diddocuments.md:226]: The `did:webs` at the start of the line should not have a capital “D”, and should probably be in back ticks.

Comment (2byrds) [reply spec/diddocuments.md]: Good eye, fixed in https://github.com/trustoverip/tswg-did-method-webs-specification/pull/59/commits/179f3d68125617c6564f296b80a964bd5a4526d7

**Pull request merged**